backtrace-library/src/main/java/backtraceio/library/metrics/UniqueEventsPayload.java

```diff
@@ -0,0 +1,21 @@
+package backtraceio.library.metrics;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.concurrent.ConcurrentLinkedDeque;
+
+import backtraceio.library.models.json.BacktraceAttributes;
+
+public final class UniqueEventsPayload extends EventsPayload {
+    @SerializedName("unique_events")
+    private ConcurrentLinkedDeque<Event> uniqueEvents;
+
+    protected UniqueEventsPayload(BacktraceAttributes backtraceAttributes, ConcurrentLinkedDeque<Event> events, int droppedEvents) {
+       super(backtraceAttributes, droppedEvents);
+       this.uniqueEvents = events;
+    }
+
+    public ConcurrentLinkedDeque<Event> getEvents() {
```

Comment:
interface

Reply:
ConcurrentLinkedDeque is a generic collection @vlussenburg :
https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/ConcurrentLinkedDeque.html

Reply:
I'm marking this resolved @vlussenburg